### ISSUE TITLE
Types of Data: Tagging datasets

### ISSUE THREAD

**Issue by LinguList** [glottobank/lexibank-data#98]
I'd propose, given issue #97 that we start thinking of tagging datasets by some simple tags. I'm thinking of (but this is only preliminary and we need to discuss it further):

* etyma: means, one *etymon* with reflexes in several languages, no strict meaning across all reflexes
* cognate sets: it's clear, but again, cognate sets does not require meaning identity
* lexicostatistic dataset: classical Swadesh-lists, *semantically aligned*
* alignments
* proto-forms

There are potentially more, and the tags I suggested are by no means consistent, but the fact that we have datasets like bdpa-germanic vs. Kitchensemitic shows that we need to have some tags to distinguish these

**Comment by xrotwang** [comment]
I think tags like "etyma" or "lexicostatistic" or "proto-forms" are useful. But we should avoid potentially redundant information like "cognate sets" or "alignments". The existence of these should be inferred from the data, no?

**Comment by xrotwang** [comment]
I'd propose we start out with a prose description for each dataset (like you did for the Concepticon concept lists), and distill useful tags from these descriptions and actual use cases - e.g. cases where the distinction between "etyma" and "lexicostatistic" becomes important.

**Comment by LinguList** [comment]
yep, I agree. BTW: we should think of fixing those column names. I'm being inconsequent myself, and other people are also inconsequent. So far, I use:

* segments for those things that are "clpa"-like or go into clpa-evalution
* cognacy for cognate sets
* alignment for alignments

But, for example, partial cognates, alignments reflecting partial cognates, etc. are not included here. Note that partial alignments and normal alignments are not the same, unless we infer normal cognates from partial cognates using the strict identity criterion. There are surely more things which will be used in a more regular fashion, and we should try to fix those names,  ideally officially in the code, or in CLDF.

**Comment by LinguList** [comment]
Good idea, regarding the prose description. One question only: writing this into the json-file is a bit tedious, and it would be easier to have a description.md, where we can use links, like we do in Concepticon, e.g., referencing a dataset, saying that this is the original one, and maybe also linking bibliographic entries to some official reference catalogue, as I do with evobib in Concepticon (but I'd also be fine linking to glottolog instead). The advantage is that we produce publication-like descriptions of data and do not really need to care of keeping a full bibliography.

**Comment by xrotwang** [comment]
Regarding the description: We already have the `NOTES.md` files incorporated in the workflow. So I guess the description should go into these.

**Comment by LinguList** [comment]
Yes, you are completely right! And I suppose from clld we can have similar functionality with the `[something](:ref:dataset)`, right?

**Comment by xrotwang** [comment]
You mean the web app could render the markdown as HTML? yes.

**Comment by LinguList** [comment]
more specifically: the :ref: inside the link would automatically translate this to the url needed, as we do with `[conceptlist](:ref:Swadesh-1952-200)`.

**Comment by xrotwang** [comment]
I wasn't aware of this. Are we doing this at http://concepticon.clld.org? For lexibank datasets I think this is not necessary, because we could use "real" relative links like
```
[other dataset](../<ds-id>/README.md)
```
upon import into the web app these could be rewritten to link to a dataset's page in the app.

**Comment by LinguList** [comment]
Also straightforward. And yes, we do it at concepticon.clld.org, this saved me a lot of time, I was also thinking of having another shortcut, like `[Anonymous (2000)](:bib:Anonymous2000)`, which would spare rewriting the links to the bibliography all the time. So far I spell those links out.

**Comment by xrotwang** [comment]
sounds like an issue for https://github.com/clld/concepticon

**Comment by xrotwang** [comment]
got it: https://github.com/clld/concepticon/issues/24

**Comment by LinguList** [comment]
yep, is this difficult to implement? If not, it would as well be useful for LexiBank's notes.md, but we need to decide on the bibliography to link to. Disadvantage of glottolog I see is that the IDs are not conveniently memorizable, and AuthorYear is pretty straightforward, as we also used it in most latex texts, etc.

**Comment by xrotwang** [comment]
For lexibank, I don't see much alternative to Glottolog - unless we want to curate an aggregated bibliography like we do for Concepticon. And for Glottolog, AuthorYear IDs for references won't work because of the number of refs.

**Comment by LinguList** [comment]
Well, then I'd vote for our curated bibliography, or the use of EvoBib. The problem is, that it will cost a tremendous amount of time to search for references via the web-interface, then pasting their number. On the other hand, having the freedom to write references in the notes.md will make a good impression on classical linguists, I suppose, as it shows that we treat the data with some scholarship in background, it shows -- at least for me -- that we take some care in writing up the descriptions.

**Comment by xrotwang** [comment]
How is EvoBib curated?

**Comment by LinguList** [comment]
well, there's some github, but I usually just take my current bibliography, which steadily gets bigger, and convert it to normal bibtex by kicking out biblatex fields. This is then converted to sqlite, uploaded, and read by the php-script.

**Comment by xrotwang** [comment]
Ok. In so that would not really be a mechanism available to other lexibank contributors. In that case, I'd say we add a lexibank.bib here.

**Comment by LinguList** [comment]
Yes, and we could, maybe, start by taking my current bibliography and just converting it to a more straightforward bibtex, that is, merging translations and Chinese texts, which are in separate columns by now.

**Comment by LinguList** [comment]
As I usually keep track of data we add to lexibank, most of the sources should already be there. Or would that be a bad idea to have too many references at once, that, is, most of them redundant?

**Comment by xrotwang** [comment]
No, I'd think this should be a good starting point.

**Comment by LinguList** [comment]
I might find time to take care of it already today.
